### ISSUE TITLE
switch to miniconda? minimamba?

### ISSUE THREAD

**Issue by rbavery** [carpentries-incubator/geospatial-python#155]
miniconda for quicker install

**Comment by mfisher87** [comment]
Have you considered [miniforge](https://github.com/conda-forge/miniforge)? It's miniconda but pre-configured to work with conda-forge, and with optional mamba support. Might be worth briefly explaining in these materials why conda-forge has value over the default channels!

**Comment by rbavery** [comment]
Agreed on explaining more about conda-forge thanks for the tips! I'm interested to try miniforge for this lesson, though I've heard anecdotal reports that recent, particular versions of mamba can conflict with some windows setups. I think we'll also always need to point to a graphical installer like Anaconda since these workshops are geared towards folks who may be new to python.

**Comment by mfisher87** [comment]
Any time! I'm really liking the materials you've put together. I recently ran a workshop with a colleague in a similar vein and will be [adding this course to our external links page](https://github.com/qgreenland-workshop-2023-researcher/qgreenland-workshop-2023-researcher.github.io/issues/169)!

Miniforge does have a nice graphical installer for Windows, and like Anaconda comes with a "Miniforge prompt" for Windows. The variant of Miniforge that includes Mamba is called "Mambaforge" and is a separate download.

Hopefully those Windows issues get fixed soon, though! I like Mamba a lot, but yeah... it's not always easy to explain the difference between it and Conda to new Python programmers.

**Comment by rbavery** [comment]
Hey cool! We're looking for other groups that want to teach this and to record if it happens so that this workshop can be certified as an official lesson cc @tobyhodges. if you all decide to teach it can you open an issue describing how it went?

Good to know about the graphical installer! I'll give that a look later.

My schedule for the next two weeks is stacked ahead of a month and a half of travel  so apologies if I don't get to any updates here in the near future.

**Comment by mfisher87** [comment]
>We're looking for other groups that want to teach this and to record if it happens so that this workshop can be certified as an official lesson cc @tobyhodges. if you all decide to teach it can you open an issue describing how it went?

Unfortunately, we're not going to have the funding (under our current grant) to deliver any more training. I'll just be sharing the materials with the participants of our previous workshops, not teaching it.

>My schedule for the next two weeks is stacked ahead of a month and a half of travel so apologies if I don't get to any updates here in the near future.

No problem at all! Looking forward to how this develops. I think the `carpentries-incubator` concept for community-developed lessons is so cool :sunglasses: 

I hope you have safe and fun travel :)
